<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessPointConfigurationWidget</class>
 <widget class="QWidget" name="AccessPointConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="accesspoint_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>780</width>
     <height>394</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>259</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>259</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>MAC Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>X Coordinate</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y Coordinate</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="filename_lineedit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>520</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish Configuration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>771</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the X and Y coordinates of any Access Points that will be involved in the training phase. Only MAC addresses which have been assigned coordinates will be used to train the system.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="selectfile_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>520</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="width_lineedit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="length_lineedit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>460</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>438</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Please specify the size of the area you will train the system on.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>505</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select (or create) a file to save this configuration to.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Measurement Spacing:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="spacing_lineedit">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>460</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QPushButton" name="add_accesspoint_button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New AccessPoint</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>width_lineedit</tabstop>
  <tabstop>length_lineedit</tabstop>
  <tabstop>spacing_lineedit</tabstop>
  <tabstop>filename_lineedit</tabstop>
  <tabstop>finish_button</tabstop>
  <tabstop>selectfile_button</tabstop>
  <tabstop>accesspoint_table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
